site/docs/contribute/environment.md

```diff
@@ -46,3 +46,53 @@ a docker registry as a proxy for all public images. This registry runs in a dock
 export DOCKERHUB_USERNAME='your docker hub username'
 export DOCKERHUB_PASSWORD='your password'
 ```
+
+## Optional Code Improvement and Verification Tools
+
+Fybrik repositories use different linters to validate and improve code.  
+*golangci-lint* works as a Go linters aggregator, which includes a lot of linter such as `staticcheck`, `revive`, `goimports` and more.  
+You can check the configuration to each repository in `.golangci.yml`.
+
+
+## Integrating golangci-lint with VS Code
+Change the default lint to golanci-lint in VS Code:
+
+1. Install golangci-lint: [https://golangci-lint.run/usage/install/](https://golangci-lint.run/usage/install/)
+2. Open VS Code `setting.json`:
+    1. Open the Command Palette: `Ctrl+Shift+P` 
+    2. Search for "Open Settings (JSON)"
+    3. Open `setting.json`
+3. Add to `setting.json` the following:
+```
+"go.lintTool":"golangci-lint",
+"go.lintFlags": [
+  "--fast"
+  "--allow-parallel-runners"
+]
+```
+
+Golangci-lint automatically discovers `.golangci.yml` config for edited file: you don't need to configure it in VS Code settings.
```

Comment:
Golangci-lint automatically discovers `.golangci.yml`  in the working project, you don't need to configure it in VS Code settings.